grafana/dashboards/timeline.json

```diff
@@ -0,0 +1,718 @@
+        "definition": "SELECT name AS __text, id AS __value FROM cars;",
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": true,
+        "label": "Car",
+        "multi": false,
+        "name": "car_id",
+        "options": [],
+        "query": "SELECT name AS __text, id AS __value FROM cars;",
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 5,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "allValue": null,
+        "current": {
+          "selected": false,
+          "text": "http://localhost:4000",
+          "value": "http://localhost:4000"
+        },
+        "datasource": null,
+        "definition": "select base_url from settings limit 1;",
+        "description": null,
+        "error": null,
+        "hide": 2,
+        "includeAll": false,
+        "label": null,
+        "multi": false,
+        "name": "base_url",
+        "options": [],
+        "query": "select base_url from settings limit 1;",
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "allValue": null,
+        "current": {
+          "selected": true,
+          "tags": [],
+          "text": [
+            "All"
+          ],
+          "value": [
+            "$__all"
+          ]
+        },
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": true,
+        "label": "Action",
+        "multi": true,
+        "name": "action_filter",
+        "options": [
+          {
+            "selected": true,
+            "text": "All",
+            "value": "$__all"
+          },
+          {
+            "selected": false,
+            "text": "🚗 Drive",
+            "value": "🚗 Drive"
+          },
+          {
+            "selected": false,
+            "text": "🔋 charges",
+            "value": "🔋 charges"
+          },
+          {
+            "selected": false,
+            "text": "🅿️ Parking",
+            "value": "🅿️ Parking"
+          },
+          {
+            "selected": false,
+            "text": "❓ Missing",
+            "value": "❓ Missing"
+          },
+          {
+            "selected": false,
+            "text": "💾 Update",
+            "value": "💾 Update"
+          }
+        ],
+        "query": "🚗 Drive,🔋 charges,🅿️ Parking,❓ Missing,💾 Update",
+        "queryValue": "",
+        "skipUrlSync": false,
+        "type": "custom"
+      },
+      {
+        "current": {
+          "selected": false,
+          "text": "",
+          "value": ""
+        },
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "label": "Text Filter",
+        "name": "text_filter",
```

Comment:
Filtering addresses does not work. It matches the strings `'Start Address'` etc instead of the columns :)

Reply:
Hmmm, no idea currently, how to do it. Disabled for now.